x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/async/StoredAsyncResponse.java

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-package org.elasticsearch.xpack.eql.async;
+package org.elasticsearch.xpack.ql.async;
```

Comment:
I would keep this generic and move it into org.elasticsearch.xpack.core.async so ML could use it. The same for StoredAsyncTask and

Reply:
Thanks, @imotov!
Will wait one more review and depending on the amount of changes required will move the classes either part of this, or of a next PR that I have to open anyways.

Reply:
The two classes have been moved.